.env_SAMPLE

```diff
@@ -0,0 +1,11 @@
+//export WORDPRESS=http://wordpress.domain
+//export GOVDELIVERY_BASE_URL=https://govdelivery.domain
+//export GOVDELIVERY_ACCOUNT_CODE=GODELIVERY_ACCT
+//export GOVDELIVERY_USER=godelivery@user.acct
+//export GOVDELIVERY_PASSWORD=P@$$w0rd
+//export SUBSCRIPTION_SUCCESS_URL=/govdelivery-subscribe-path
+//export SUBSCRIPTION_USER_ERROR_URL=/govdelivery-subscribe-error-path
+//export SUBSCRIPTION_SERVER_ERROR_URL=/govdelivery-subscribe-server-error-path
+//echo 'Environment variables for WORDPRESS, GOVDELIVERY, and SUBSCRIPTION are set'
```

Comment:
The SUBSCRIPTION settings appear to be a part of GovDelivery, so this would just be envvars set for WP + GovDelivery. Also, this echo is rather unreliable without any logic to check whether the envvars have in fact been set.